Intersect.Editor/Forms/Editors/Events/Event Commands/EventCommand_ConditionalBranch.cs

```diff
@@ -383,6 +387,14 @@ private void ConditionTypeChanged(ConditionTypes type)
                         cmbMapZoneType.SelectedIndex = 0;
                     }
 
+                    break;
+                case ConditionTypes.CheckEquipment:
+                    Condition = new CheckEquippedSlot();
+                    if (cmbCheckEquippedSlot.Items.Count > 0)
+                    {
+                        cmbCheckEquippedSlot.SelectedIndex = 0;
+                    }
```

Comment:
Look at this again, the `if` is not needed, just set the SelectedIndex to 0, just like on L387 (for MapZoneTypeIs)